src/components/Modal.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "@wordpress/element";
+import { ReactComponent as CloseIcon } from "../icons/close.svg";
+import { ReactComponent as Help } from "../icons/help.svg";
+import HelpCenter from "./HelpCenter";
+
+import { toggleHelp } from "..";
+import { LocalStorageUtils } from "../utils";
+
+const Modal = ({ onClose }) => {
+  useEffect(() => {
+    const helpVisible = LocalStorageUtils.getHelpVisible();
+    toggleHelp(helpVisible);
+  }, []);
+
+  return (
+    <div className="modal">
+      <div className="modal-header">
+        <h3 className="heading">
+          <span className="icon">
+            <Help />
+          </span>
+          Help Center
```

Comment:
@amartya-dev We should use the `__()` function to allow text strings like this to be translated. You'd need to import the https://www.npmjs.com/package/@wordpress/i18n package. We'd need to be sure and apply this to the other JS files with raw strings as well.